### PR TITLE
Feature/disc talents second pass

### PR DESCRIPTION
**Pull request by oisinod**

### CODE REVIEW

File: src/common/SPELLS/priest.ts [hunk 1]
```diff
@@ -73,20 +73,20 @@ const spells = spellIndexableList({
...
     icon: 'spell_holy_penance',
-    manaCost: 800,
+    manaCost: 4000,
   },
   PENANCE_CAST: {
     id: 47540,
     name: 'Penance',
     icon: 'spell_holy_penance',
-    manaCost: 800,
+    manaCost: 4000,
```
Comment (emallson) [inline src/common/SPELLS/priest.ts:89]: these mana costs should be in the talent data, right? or is penance not a talent for disc?

**Review by emallson — COMMENTED** [review]
one comment but otherwise looks good

File: src/common/SPELLS/priest.ts (hunk 1 above)
Comment (oisinod) [reply src/common/SPELLS/priest.ts:89]: It's baseline :)

Comment (emallson) [reply src/common/SPELLS/priest.ts:89]: unfortunate

**Pull request merged**